### ISSUE TITLE
Adding the possibility to change the domain of the cookie when Saml2 is configured in Startup.cs

### ISSUE THREAD

**Issue by mokv** [ITfoxtec/ITfoxtec.Identity.Saml2#92]
Hello,

In my project we need to set Domain of the Cookie. Can you add this functionality?

In file `src/ITfoxtec.Identity.Saml2.MvcCore/Configuration/Saml2ServiceCollectionExtensions.cs`, in method `public static IServiceCollection AddSaml2(this IServiceCollection services,`  add parameter `string cookieDomain = null` and in the implementation add the following:
```
if (!string.IsNullOrEmpty(cookieDomain))
{
    o.Cookie.Domain = cookieDomain;
}
```

There might be something I am not aware of, so please advice if this would be a problem to add.

Thank you in advance.

**Comment by Revsgaard** [comment]
Thank you.
I have added your suggested functionality in the https://github.com/ITfoxtec/ITfoxtec.Identity.Saml2/commit/0e1d7d7f55db22bc423c3ed5bc993635aece416f commit.

**Comment by Revsgaard** [comment]
Released in [4.7.0](https://github.com/ITfoxtec/ITfoxtec.Identity.Saml2/releases/tag/4.7.0)

**Comment by mokv** [comment]
I appreciate the effort. Thank you.
